backend/app/views/spree/admin/images/index.html.erb

```diff
@@ -5,7 +5,9 @@
 
 <% content_for :page_actions do %>
   <% if can?(:create, Spree::Image) %>
-    <li><%= link_to_with_icon('plus', t('spree.new_image'), new_admin_product_image_url(@product), id: 'new_image_link', class: 'btn btn-primary') %></li>
+    <li id="new_image_link">
+      <%= link_to t('spree.new_image'), new_admin_product_image_url(@product), id: 'new_image_link', class: 'btn btn-primary' %>
```

Comment:
Looks like you've applied the `new_image_link` ID to both the `li` and the `a` elements. Since IDs should be unique and it's conceivable that someone could be using deface overrides to target this, I think it's best to leave the ID on the `a` and not also add it to the `li`.

Reply:
@jarednorman Good catch!

Looking around at other backend views, it seems like in https://github.com/solidusio/solidus/blob/master/backend/app/views/spree/admin/prices/index.html.erb#L5-L7 and https://github.com/solidusio/solidus/blob/master/backend/app/views/spree/admin/variants/index.html.erb#L7-L13, the convention is to put the ID on the `li` element. I’d suggest that we do the same here for consistency with those other views.

I’ve force-pushed the latest; LMK what you think.